src/App.vue

```diff
@@ -557,7 +560,7 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
           await this.validateReviewCertifyPage()
         } else {
           // Check validity, and if OK then go to summary page.
-          await this.validateCompanyInfoPage()
+          await this.onClickSave()
```

Comment:
onClickSave() is what directs to the pay URL but I'm not sure if I am still supposed to validate the company info page

Reply:
This actually breaks the review and confirm button

Reply:
As this change is in App.vue, it will affect all filings (views) in this UI.

I think you should revert this change. What are you trying to do here? (We can have a quick chat if you want.)

Reply:
Obsolete code.